tests/modules/bodfeedback/test_bodfeedback.py

```diff
@@ -0,0 +1,172 @@
+import flask
+from donut.modules.bodfeedback import helpers
+from donut.testing.fixtures import client
+from datetime import datetime
+
+
+def test_get_link(client):
+    assert helpers.get_link(
+        1) == flask.url_for('home') + 'bodfeedback/view/sample_uuid'
```

Comment:
Can't this be done with a `flask.url_for()` call rather than building the path yourself?